### ISSUE TITLE
Update docs

### ISSUE THREAD

**Issue by ElliottKasoar** [stfc/aiida-mlip#156]
Another pass over the docs would be good. A few things I've already picked up include:

In [getting started](https://stfc.github.io/aiida-mlip/user_guide/get_started.html#getting-started):

- [x] `#pip install -e .[pre-commit,testing] # install extras for more features` needs updating to work with poetry, and probably doesn't need to be in the quick start at all, but rather in the dev guide
- [x] `verdi quicksetup` is deprecated
- [x] The full docstring for `Singlepoint` makes is a bit unreadable
- [ ] `examples/test_submit` has moved/doesn't exist?
- [ ] More specific instructions (here or in the tutorial about adding a computer/code), particularly in the context of having `janus` installed

Developer guide

- [x] Update to match janus-core e.g. correct info for installation via poetry, or otherwise
- [x] We can probably remove publishing and release

Tutorial

- [ ] `Install aiida-mlip, [aiida-core](https://aiida.readthedocs.io/projects/aiida-core/en/latest/intro/get_started.html), [janus-core](https://github.com/stfc/janus-core).` could probably include a link, and/or clarify that installing aiida-mlip should install the requirements anyway

Citation

- [ ] We currently have "Author Name1, Author Name2, Paper title, Jornal Name XXX, YYYY (Year)."
- [ ] We should add a CITATION.cff - see janus-core for example

Other

- [x] I suggest we can probably drop the full repository contents in the README

**Comment by ElliottKasoar** [comment]
@MM0hsin can you check these off and confirm when everything is done (if you have edit permission, you can click the boxes and they tick)? The `Citation` parts are not, but your changes in #178 might have covered everything else.

**Comment by ElliottKasoar** [comment]
> > More specific instructions (here or in the tutorial about adding a computer/code), particularly in the context of having janus installed
> 
> This would be great to have because as a user I'm currently not sure if I'm configuring the Code correctly. Should the `filepath executable` be Bash? Python? `janus`? How does it work if Janus and aiida are installed in a virtual environment together? Can they be installed in different Python environments?
> 
> Pointing it to `my_project/.venv/bin/janus` seems to work but I don't know if that's just by good fortune. Would it also work to do things like `uv run --project /path/to/other/project janus`?
> 
> I guess the underlying non-obvious thing is: does the plugin interact with the `janus` CLI, the Python interface, or a mixture of both?

Good questions!

You've just helped me realise that the latest documentation updates didn't trigger a rebuild, so things should look a little better once another PR is merged. We did make a bit of an effort to clarify the code/computer requirements, but still not in as much detail as you've suggested.

I'm also a bit hesitant to repeat too much of AiiDA's own documentation, so we try to link things where relevant, e.g.  https://aiida.readthedocs.io/projects/aiida-core/en/stable/howto/run_codes.html#how-to-create-a-code, as well as our own tutorials e.g. https://github.com/stfc/aiida-mlip/blob/main/examples/tutorials/setup-janus-code.ipynb (although it sounds like you found this anyway?).

Obviously it's hard to get a complete picture without seeing the links in the context of the updated docs, and the notebook definitely could use another pass, but between those links, it doesn't feel too ambiguous to me than the `janus` `Code` should be a `janus` executable?

"Which executable" is a slightly tricker question. The expectation is definitely that you'd use the one that is installed with `aiida-mlip`. I think it would "work" with any other installation too that would launch the CLI if you ran it as a command, including your `uv run` example, but I'd imagine there's a chance you run into dependency conflicts.

I believe the only interface is to the `janus` CLI (although ultimately that is passed on to the Python interface). We should clarify that somewhere.

**Comment by ajjackson** [comment]
I'll admit I didn't think to look at the .ipynb tutorial for configuration instructions! Yes, that block makes it clear that `janus` is the only executable to configure, and I think would be worth having in the main docs.

The main appeal of `uv run` is to _avoid_ dependency issues where the `aiida-mlip`/`janus` stack needs one set of pinned libraries but in the same AiiDA session one might like to use some plugins that have conflicting requirements. If the plugin only interacts with janus via the shell interface, such things could be fairly safe to do, which is nice!

But it's a bit surprising that a Python package (aiida-mlip) would install another Python package (janus-core) as a Python dependency and then _not_ communicate with it via Python. Unless this is explicitly promised somewhere, perhaps it is not advisable to rely on that behaviour!
